### PR TITLE
Docs: New Motion Guidelines page

### PR DESCRIPTION
**Pull request by diyorbek**
### Summary

New motion guidelines pages.

#### What changed?

1. Ranamed the section "Animation" to "Motion"
2. Deleted "Implementation" page. It will be replaced with new implementation page once Motion tokens are ready.

### Links

- [Jira](https://jira.pinadmin.com/browse/GESTALT-7685)

### Checklist

- [ ] Added unit and Flow Tests
- [ ] Added documentation + accessibility tests
- [x] Verified accessibility: keyboard & screen reader interaction
- [x] Checked dark mode, responsiveness, and right-to-left support
- [ ] Checked stakeholder feedback (e.g. Gestalt designers, relevant feature teams)

### CODE REVIEW

File: docs/pages/foundations/motion/guidelines.tsx [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+`}
+          />
+        </Box>
+      </MainSection>
+
+      <MainSection
+        description={`
+        Animations should never cause a user harm or distract from a necessary task. However, inaccessible animations have the potential to cause seizures, motion sickness, or anxiety in users, so we must ensure we are creating accessible animations every time.
+
+        Below are 3 key considerations for accessibility. Note: these do not apply for “essential” animations, where [essential is defined](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html#essentialdef) as any animation where “if removed, would fundamentally change the information or functionality of the content, _and_ information and functionality cannot be achieved in another way that would conform.”
```
Comment (danhendersonede) [inline docs/pages/foundations/motion/guidelines.tsx:248]: This is referring to the WCAG 2.0 guidelines, let's update to WCAG 2.2: https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide.html#dfn-essential

File: docs/pages/foundations/motion/guidelines.tsx [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+        name="Accessibility"
+      >
+        <MainSection.Subsection
+          columns={2}
+          description={`
+          Ensure that any animation that is moving, blinking, scrolling or auto-playing and lasts longer than 5 seconds has an affordance for pausing and resuming the animation or an option to hide the animation.
+
+          _Looping animations should be avoided whenever possible, as they can become distracting very quickly._
+
+          Further reading: [WCAG 2.0 Pause, Stop Hide requirements](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html)
```
Comment (danhendersonede) [inline docs/pages/foundations/motion/guidelines.tsx:259]: Let's update to 2.2

File: docs/pages/foundations/motion/guidelines.tsx [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+          `}
+          title="Looping animations"
+        />
+        <MainSection.Subsection
+          columns={2}
+          description={`
+          Ensure that there are no flashing animations present, where flashing is defined as anything that flashes _more than three times in a one second period_.
+
+          Further reading: [WCAG 2.0 Three Flashes requirements](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-three-times.html)
```
Comment (danhendersonede) [inline docs/pages/foundations/motion/guidelines.tsx:268]: Let's update to 2.2

File: docs/pages/foundations/motion/principles.tsx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+                />
+              </Box>
+            </Flex.Item>
+            <Flex.Item flex="grow" flexBasis={240}>
+              <Box borderStyle="sm" height="100%" padding={4} rounding={2}>
+                <PrincipleItem
+                  heading="Accessible"
+                  text={
+                    <Text>
+                      Animations must be implemented in a way that maintains accessibility, ensuring
```
Comment (danhendersonede) [inline docs/pages/foundations/motion/principles.tsx:66]:
```suggestion
                      Animations must be designed and implemented to consider people with disabilities, ensuring
```

File: docs/pages/foundations/motion/guidelines.tsx [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+          `}
+          title="Flashing animations"
+        />
+        <MainSection.Subsection
+          columns={2}
+          description={`
+          Any non-essential animations should automatically be disabled when a user enables the reduced motion setting on their device. For Web, we have provided a [useReducedMotion hook](https://gestalt.pinterest.systems/web/utilities/usereducedmotion) to make this easy to implement.
+
+          Further reading for Web: [MDN prefers-reduced-motion guidelines](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
+          Further reading for iOS: [Apple developer guidelines](https://developer.apple.com/documentation/uikit/uiaccessibility/1615133-isreducemotionenabled)
```
Comment (danhendersonede) [inline docs/pages/foundations/motion/guidelines.tsx:278]: Here's some good guidelines for Android: https://appt.org/en/docs/android/samples/reduced-animations 

Here's some better guidelines for iOS: https://appt.org/en/docs/ios/samples/reduced-animations

**Comment by danhendersonede** [comment]
There's also lots of images and gifs on this page without alt text. We should add a descriptions to many of these items.

**Comment by danhendersonede** [comment]
Also, the motion guidelines page doesn't actually follow our own accessibility guidelines for motion :grimacing:

See: https://css-tricks.com/gifs-and-prefers-reduced-motion/

File: docs/docs-components/data/foundations.tsx [hunk 6]
```diff
@@ -25,10 +25,10 @@ const foundationsTiles: ReadonlyArray<DesignOverview> = [
     path: '/foundations/accessibility',
   },
   {
-    svg: <Animation />,
-    title: 'Animation',
+    svg: <Motion />,
+    title: 'Motion',
     description: 'Guidelines for animation in product and implementation options',
-    path: '/foundations/animation/principles',
+    path: '/foundations/motion/principles',
```
Comment (AlbertCarreras) [inline docs/docs-components/data/foundations.tsx:31]: can you add this change to the redirect section? @diyorbek

File: docs/pages/foundations/motion/guidelines.tsx [hunk 7]
```diff
@@ -0,0 +1,360 @@
...
+          `}
+          title="Flashing animations"
+        />
+        <MainSection.Subsection
+          columns={2}
+          description={`
+          Any non-essential animations should automatically be disabled when a user enables the reduced motion setting on their device. For Web, we have provided a [useReducedMotion hook](https://gestalt.pinterest.systems/web/utilities/usereducedmotion) to make this easy to implement.
+
+          Further reading for Web: [MDN prefers-reduced-motion guidelines](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
+          Further reading for iOS: [Appt iOS guidelines](https://appt.org/en/docs/ios/samples/reduced-animations)
```
Comment (AlbertCarreras) [inline docs/pages/foundations/motion/guidelines.tsx:352]: [Appt _**reduced-animations**_  Android guidelines]

Comment (AlbertCarreras) [reply docs/pages/foundations/motion/guidelines.tsx:352]: [Appt _**reduced-animations**_ iOS guidelines]

**Review by AlbertCarreras — APPROVED** [review]
LGTM! There's all those images not showing. I added a few minor comments

File: playwright/accessibility/foundations__animation__implementation.spec.mjs [hunk 8]
```diff
@@ -1,8 +0,0 @@
-// @flow strict
-import { test } from '@playwright/test';
-import expectAccessiblePage from './expectAccessiblePage.mjs';
-
```
Comment (AlbertCarreras) [inline playwright/accessibility/foundations__animation__implementation.spec.mjs:4]: Screenshot by Dropbox Capture

**Pull request merged**